site/src/components/Abbr/Abbr.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * @file A more sophisticated version of the native <abbr> element.
+ *
+ * Features:
+ * - Better type-safety (requiring you to include certain properties)
+ * - All built-in HTML styling is stripped away by default
+ * - Better integration with screen readers, with more options for controlling
+ *   how they read out initialisms.
+ */
+import { visuallyHidden } from "@mui/utils";
+import { type FC, type HTMLAttributes } from "react";
+
+// There isn't a dedicated HTMLAbbrElement type; have to go with base type
+type AbbrProps = Exclude<HTMLAttributes<HTMLElement>, "title" | "children"> & {
+  // Not calling this "title" to make it clear that it doesn't have the same
+  // issues as the native title attribute as far as screen reader support
+  expandedText: string;
+  children: string;
+  initialism?: boolean;
```

Comment:
NIT: what do you think about the var names "tooltipText" (instead of "expandedText") and "hasAcronym" (instead of "initialism")? Expanded makes me think there is a dropdown or accordion and I had to google _initialism_.

Reply:
Ah, thanks for those distinctions; makes sense. I like the proposal you came up with, i.e. 
`pronunciation?: "shorthand" | "acronym" | "initialism";`